app/build.gradle

```diff
@@ -34,25 +34,27 @@ android {
                     'proguard-rules-retrofit.pro'
         }
     }
+    dynamicFeatures = [':paintings']
```

Comment:
`app` is the base feature module. It needs to declare all the dynamic feature modules that can be installed on top of it.

In this PR, we've added a single dynamic feature module: `paintings`.